spec/amp-localstorage.md

```diff
@@ -58,6 +58,7 @@ The following AMP components and service are using the localStorage.
 -   `<amp-user-notification>` : Store the user decision on dismiss notification
 -   `<amp-consent>` : Store the user decision and granular information on consent
 -   Client ID Service : Store the user decision to opt-out CID service.
+-   Consent Instance : Store the user consent decision.
```

Comment:
Is this different from the above:
```
<amp-consent> : Store the user decision and granular information on consent
```

Reply:
Added it because Consent Instance is a service. And defined in `consent-state-manager.js` not in `amp-consent.js`.